util.go

```diff
@@ -0,0 +1,58 @@
+package lo
+
+// Range creates an array of numbers (positive and/or negative) with given length.
+func Range(elementNum int) []int {
+	if elementNum >= 0 {
```

Comment:
Range can be simplified into:

```go
func Range(start, elementNum int) []int {
	result := make([]int, If(elementNum < 0, -elementNum).Else(elementNum))
	it := If(start <= elementNum, 1).Else(-1)

	for j, i := 0, start; i != elementNum; j, i = j+1, i+it {
		result[j] = i + start
	}

	return result
}
```

Reply:
Ogg. Yeah, I forgot there is a condition function that can be used..